.github/workflows/deploy.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # make sure we have version info
+      - run: git fetch --tags
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
why 3.8? in lstchain we are still requiring 3.7

Reply:
Doesn't matter at all, since all it does is creating the tar balls.

Reply:
>  in lstchain we are still requiring 3.7

Because protozfits only worked with 3.7, now you could change that ;)